vignettes/rd.Rmd

```diff
@@ -66,7 +66,7 @@ add(10, 1)
 
 Rd files are a special file format loosely based on LaTeX. You can read more about the Rd format in the [R extensions](https://cran.r-project.org/doc/manuals/R-exts.html#Rd-format) manual. With roxygen2, there are few reasons to know about Rd files, so here I'll avoid discussing them as much as possible, focussing instead on what you need to know about roxygen2.
 
-When you use `?x`, `help("x")` or `example("x")` R looks for an Rd file containing `\alias{x}`. It then parses the file, converts it into html and displays it. These functions look for an Rd file in _installed_ packages. This isn't very useful for package development, because you want to use the `.Rd` files in the _source_ package. For this reason, we recommend that you use roxygen2 in conjunction with devtools:  `dwevtools::load_all()` automatically adds shims so that `?` and friends will look in the development package. Note, however, that this preview does not work with intra-package links. To preview those, you'll need to install the package. If you use RStudio, the easiest way to do this is to click the "Build & Reload button".
+When you use `?x`, `help("x")` or `example("x")` R looks for an Rd file containing `\alias{x}`. It then parses the file, converts it into html and displays it. These functions look for an Rd file in _installed_ packages. This isn't very useful for package development, because you want to use the `.Rd` files in the _source_ package. For this reason, we recommend that you use roxygen2 in conjunction with `devtools::load_all()` automatically adds shims so that `?` and friends will look in the development package. Note, however, that this preview does not work with intra-package links. To preview those, you'll need to install the package. If you use RStudio, the easiest way to do this is to click the "Build & Reload button".
```

Comment:
You also removed the word "devtools". I think  this should read as:

> in conjunction with devtools:  `devtools::load_all()` automatically

Reply:
gotcha, sorry, done.